### ISSUE TITLE
Bug - Truncate/Progress - truncated content cannot be focused via keyboard

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11384]
**Describe the problem**
A clear and concise description of the problem. Which components are affected?

In the Truncate and Progress (truncated example) component, the truncated content cannot be focused via keyboard which would be necessary to trigger a tooltip to display the entire content. See how Alert handles truncation in its title:

https://github.com/patternfly/patternfly-react/blob/a2cb15519172f33cc701fd6b6cc5e7405a6d8a4c/packages/react-core/src/components/Alert/Alert.tsx#L198-L201

**How do you reproduce the problem?**
Provide steps to reproduce. A [codesandbox](https://codesandbox.io/s/serverless-cherry-q9t3f ) demonstrating the problem is appreciated.

Go to the [Truncate examples](https://www.patternfly.org/components/truncate) and the [truncate title progress example](https://www.patternfly.org/components/progress#truncate-title) and try tabbing through the page. Notice how the truncated content never receives focus to trigger a tooltip.

**Expected behavior**
A clear and concise description of the expected behavior.

The truncated content would receive focus and trigger the tooltip

**Is this issue blocking you?**
List the workaround if there is one.

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**

**What is your product and what release date are you targeting?**

**Any other information?**

**Comment by thatblindgeye** [comment]
We should look into whether we have an established pattern for focusing truncated content elsewhere. If not, utilizing tabIndex could be a solution to this.
